### PR TITLE
Upgrade outputs ODV

### PR DESCRIPTION
**Pull request by Marie59**
I added a few lines in the wrapper in the hope to be able to get ODV's outputs in the galaxy history (https://github.com/usegalaxy-eu/galaxy/issues/170).
Indeed, for now I can't use ODV outputs and use them as inputs in other tools. Are the changes that I made relevant @bgruening ? Sorry if I am totally lost here.

### CODE REVIEW

**Comment by bgruening** [comment]
Do you know why you need to be in the output directory when starting the application?

**Comment by Marie59** [comment]
> Do you know why you need to be in the output directory when starting the application?

No need. That's not what I was aiming at doing it's an useless line that I wrongly copy paste from my test I think. From what I understood (which might be not that much) the outputs folder in the ODV app is not link to the Galaxy history outputs. Is the cp command enough to do so ?

File: tools/interactive/interactivetool_odv.xml [hunk 1]
```diff
@@ -17,6 +17,7 @@
     	mkdir output &&
         mkdir -p /root/Documents/ODV/data/output &&
         cp '$infile' '/root/Documents/ODV/data/$infile.display_name' &&
+        cp /root/Documents/ODV/data/output/* output/ | true &&
         bash /startup.sh
```
Comment (bgruening) [inline tools/interactive/interactivetool_odv.xml]:
```suggestion
    	ln -s  /root/Documents/ODV/data/output/ ./output/ &&
        mkdir -p /root/Documents/ODV/data/output &&
        cp '$infile' '/root/Documents/ODV/data/$infile.display_name' &&
        bash /startup.sh
```

Comment (bgruening) [reply tools/interactive/interactivetool_odv.xml]: Do you think this might work as well?

**Comment by Marie59** [comment]
Oh okay that's how I can do it ! Thanks !!

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@Marie59 can you check if this is working now?

**Comment by Marie59** [comment]
It's not working I have this error `ln: target './output/' is not a directory: No such file or directory`. Could it be because now there is'nt this lin in the xml  `mkdir output &&` ?

**Pull request merged**